great_expectations/util.py

```diff
@@ -119,6 +119,10 @@ def underscore(word: str) -> str:
     return word.lower()
 
 
+def hyphen(input: str):
```

Comment:
Why is this function needed?

Reply:
used to convert between python naming conventions and URL namespaces. keeping it here could go either way, but I felt it belonged alongside the existing `underscore` method, and avoid repeating the code in two places in case we need special behavior